src/virtualenv.py

```diff
@@ -1893,13 +1890,11 @@ def after_install(options, home_dir):
     with codecs.open(filename, "r", encoding="utf-8") as f:
         content = f.read()
     py_exe = "python%s" % python_version
-    content = ("#!/usr/bin/env %s\n" % py_exe) + "## WARNING: This file is generated\n" + content
+    content = ("#!/usr/bin/env %s\n" % py_exe) + "# WARNING: This file is generated\n" + content
     return content.replace("##EXT" "END##", extra_text)
 
 
-##EXTEND##
-
-
+# EXTEND
```

Comment:
Due to this change, created bootstrap script does not have any extra_text.
https://virtualenv.pypa.io/en/latest/reference/#creating-your-own-bootstrap-scripts

With virtualenv 16.0.0
```
$ python -c "import virtualenv; print(virtualenv.create_bootstrap_script('HELLOWORLD'))"| grep HELLOWORLD
HELLOWORLD
$
```

With virtualenv 16.1.0 (same as master)
```
$ python -c "import virtualenv; print(virtualenv.create_bootstrap_script('HELLOWORLD'))"| grep HELLOWORLD
$
```
Could you fix it?

Reply:
yeah let's open an issue for it 😎

Reply:
Sorry too late. I created issue #1244